crates/weaver_forge/src/extensions/otel.rs

```diff
@@ -324,6 +325,19 @@ pub(crate) fn instantiated_type(attr_type: &Value) -> Result<String, minijinja::
     )))
 }
 
+/// Converts an enum member value into:
+/// - A quoted string if the input is a string.
+/// - A non-quoted string if the input is a number or a boolean.
+/// - An empty string otherwise.
+pub(crate) fn print_member_value(input: &Value) -> String {
+    match input.kind() {
+        ValueKind::String => format!("\"{}\"", input),
```

Comment:
Do we need to "escape" quotes in the string when doing this?  Likely...

Reply:
Oh yes. It was planned and I forgot...

Reply:
Fixed